gtsam/navigation/NavState.cpp

```diff
@@ -472,9 +484,8 @@ Vector9 NavState::coriolis(double dt, const Vector3& omega, bool secondOrder,
     D_v_v(H) << D_body_nav * (-2.0 * dt) * D_cross_state;
     D_v_R(H) << D_dV_R;
     if (secondOrder) {
-      const Matrix3 D_cross2_state = Omega * D_cross_state;
-      D_t_t(H) -= D_body_nav * (0.5 * dt2) * D_cross2_state;
-      D_v_t(H) -= D_body_nav * dt * D_cross2_state;
+      D_t_t(H) -= (0.5 * dt2) * D_body_nav * (H2_wrt_t * R());
+      D_v_t(H) -= dt * D_body_nav * (H2_wrt_t * R());
```

Comment:
dt * D_c2_np; ????

I'd add a unit test with non-identity R before - make sure it fails - then apply fix, should succeed.